common/src/main/webpack/prod.webpack.config.js

```diff
@@ -52,6 +52,9 @@ const Web = Merge(
         persistentCache: false,
       }),
     ],
+    node: {
+      fs: "empty"
+    }
```

Comment:
Is this required by the new CSS plugin?

Reply:
Not really. I'm trying to remove a warning about the fsevents dependency while trying to fix the test app issue